docs/crio.8.md

```diff
@@ -166,6 +167,8 @@ For the bind-mount conditions, only mounts explicitly requested by Kubernetes co
 
 **--log-level, -l**="": log crio messages above specified level: debug, info, warn, error (default), fatal or panic
 
+**--log-filter**="": filter the log messages by the provided regular expression
+
```

Comment:
here and at line 36, it would be nice to have in the first line '--log=[value]` and then have the rest of the `--log-*` entries listed in alphabetical order.

Reply:
Sure, I resort them. :+1: